Makefile

```diff
@@ -91,14 +91,14 @@ IMMUTABLE_DOCKER_TAG := $(VERSION)
 .PHONY: lint
 lint:
 	$(GO_DOCKER_CMD) sh -c ' \
-		cd /exporter && \
-		golangci-lint run --config ../../golangci.yaml \
+		cd exporter && \
+		golangci-lint run --config ../golangci.yaml \
```

Comment:
This path wasn't adjusted in the name change PR. Had to do this to get linter working again.